lib/fieldsets/nestedFieldset.js

```diff
@@ -277,7 +277,7 @@ const _parseRepeatName = function (string) {
 // Check helpers
 
 const _assertNotNull = function (value, msg) {
-    if (value == null) {
+    if (value === null) {
```

Comment:
This is meant to detect both `undefined` and `null`, the use of `==` is on purpose